source/extensions/filters/http/jwt_authn/authenticator.cc

```diff
@@ -0,0 +1,264 @@
+#include "extensions/filters/http/jwt_authn/authenticator.h"
+
+#include "envoy/http/async_client.h"
+
+#include "common/common/logger.h"
+#include "common/http/message_impl.h"
+#include "common/http/utility.h"
+
+#include "include/jwt_verify_lib/jwt.h"
```

Comment:
Nit: any chance we can drop the `include/` part of this? It's just aesthetics, but yeah..